modules/op-specifying-manual-approval-task.adoc

```diff
@@ -0,0 +1,59 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-specifying-manual-approval-task_{context}"]
+= Specifying a manual approval task
+
+You can specify a manual approval task in your pipeline. When the execution of a pipeline run reaches this task, the pipeline run stops and awaits approval from one or several users.
+
+.Prerequisites
+
+* You enabled the manual approver gate controller.
+* You created a YAML specification of a pipeline.
+
+.Procedure
+
+* Specify an `ApprovalTask` in the pipeline, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: example-manual-approval-pipeline
+spec:
+  tasks:
+# ...
+  - name: example-manual-approval-task
+    taskRef:
+      apiVersion: openshift-pipelines.org/v1alpha1
+      kind: ApprovalTask
+    params:
+    - name: approvers
+      value:
+      - user1
+      - user2
+      - user3
+    - name: description
+      value: Example manual approval task - please approve or reject
+    - name: numberOfApprovalsRequired
+      value: '2'
```

Comment:
When I tested I put an integer here, not a string. @PuneetPunamiya can you confirm which one is correct/if both are ok?